packages/block/src/header.ts

```diff
@@ -361,12 +427,27 @@ export class BlockHeader {
 
   /**
    * Returns the block header in JSON format
-   *
-   * @see {@link https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/index.md#defineproperties|ethereumjs-util}
    */
   toJSON(_labels: boolean = false): { [key: string]: string } | string[] {
     // Note: This never gets executed, defineProperties overwrites it.
-    return {}
+    /*return Object.create({
+      parentHash: this.parentHash,
+      uncleHash: this.uncleHash,
+      coinbase: this.coinbase,
+      stateRoot: this.stateRoot,
+      transactionsTrie: this.transactionsTrie,
+      receiptTrie: this.receiptTrie,
+      bloom: this.bloom,
+      difficulty: this.difficulty,
+      number: this.number,
+      gasLimit: this.gasLimit,
+      gasUsed: this.gasUsed,
+      timestamp: this.timestamp,
+      extraData: this.extraData,
+      mixHash: this.mixHash,
+      nonce: this.nonce,
+    })*/
+    return {} //TODO: FIXME
```

Comment:
Explicit comment that we should fix this one here since `defineProperties` does not override this anymore.

Reply:
Also not sure what this `_labels` parameters is supposed to do, is this a switch between returning either ` { [key: string]: string }` or  `string[]`?

Reply:
that param can be removed.

for an example to replicate see [transaction.toJSON()](https://github.com/ethereumjs/ethereumjs-vm/blob/master/packages/tx/src/transaction.ts#L329)

Reply:
commit incoming that will resolve this